l10nzh.properties

```diff
@@ -2,13 +2,19 @@ category=Localization
 description=SonarQube Chinese Pack
 homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zh
 archivedVersions=
-publicVersions=1.29,8.0,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9.0,9.1,9.2
+publicVersions=1.29,8.0,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9.0,9.1,9.2,9.3
```

Comment:
It's time to de-list plugin versions that support SonarQube versions < 8.9. I won't hold this release for that, but I'd like to see it handled for next time, please.